reporter/grpc_env.go

```diff
@@ -32,6 +33,8 @@ const (
 	swAgentCollectorGetAgentDynamicConfigInterval = "SW_AGENT_COLLECTOR_GET_AGENT_DYNAMIC_CONFIG_INTERVAL"
 	swAgentCollectorBackendServices               = "SW_AGENT_COLLECTOR_BACKEND_SERVICES"
 	swAgentCollectorMaxSendQueueSize              = "SW_AGENT_COLLECTOR_MAX_SEND_QUEUE_SIZE"
+	swAgentProcessStatusHookEnable                = "SW_AGENT_PROCESS_STATUS_HOOK_ENABLE"
+	swAgentProcessLabels                          = "SW_AGENT_PROCESS_LABELS"
```

Comment:
What happens if this env variable is set, but no `WithProcessLabels` called in the codes?

Reply:
It could be detected and triggered `WithProcessLabels`. please take a look on: https://github.com/SkyAPM/go2sky/pull/156/files#diff-758d4819d25e782438ebef78193df4c798a5a0b247693e5f44ae9a514e353d53R82-R93